renovate.json

```diff
@@ -55,6 +55,16 @@
       "matchStrings": [
         "- gomod: (?<depName>(go\\.opentelemetry\\.io/collector|github\\.com/open-telemetry/opentelemetry-collector-contrib|github\\.com/dynatrace/dynatrace-otel-collector)/([A-Za-z0-9]+/)*[A-Za-z0-9]+) (?<currentValue>v\\d+\\.\\d+\\.\\d+)\\n"
       ]
+    },
+    {
+      "customType": "regex",
+      "fileMatch": [
+        "^\\.github/workflows/.*\\.yaml$"
+      ],
+      "datasourceTemplate": "go",
+      "matchStrings": [
+        "run: go install (?<packageName>github\\.com/CycloneDX/cyclonedx-gomod/cmd/cyclonedx-gomod)@(?<currentValue>v\\d+\\.\\d+\\.\\d+)\\n"
+      ]
```

Comment:
we have a good method of adding regex matches to renovate already. see [this config line](https://github.com/Dynatrace/dynatrace-otel-collector/pull/541/files#diff-7b5c8955fc544a11b4b74eddb4115f9cc51c9cf162dbffa60d37eeed82a55a57R46)
and a usage of that can be found [here](https://github.com/Dynatrace/dynatrace-otel-collector/pull/541/files#diff-8da12a3c4afea00f93c8ec7c9a320c45d3209a66fefd270b60faff9728895914R12-R13)

Reply:
Thx for the hint. Unfortunately, the existing matching pattern only works for variable assignments, due to the `(:|=|\\?=|:=|\\+=)` part. 

I've now extended it by `@` to also match lines like `... cyclonedx-gomod@v1.9.0`